pkg/algorithm/cstorpoolselect/v1alpha1/select.go

```diff
@@ -103,6 +113,93 @@ type policy interface {
 	filter(*csp.CSPList) (*csp.CSPList, error)
 }
 
+// scheduleWithOverProvisioningAwareness is a pool
+// selection implementation.
+type scheduleWithOverProvisioningAwareness struct {
+	overProvisioning bool
```

Comment:
will it be possible to explain these fields. It helps to maintain & read the code.

Reply:
done